server/lib/tuist_web/live/build_run_live.html.heex

```diff
@@ -22,19 +22,31 @@
           <.alert_circle />
         </div>
       </div>
-      <h1 :if={not is_nil(@run.scheme)} data-part="label">{@run.scheme}</h1>
+      <h1 data-part="label">{@run.scheme || gettext("Unknown scheme")}</h1>
+    </div>
+    <div data-part="actions">
+      <.button
+        :if={@has_result_bundle.ok? && @has_result_bundle.result && @command_event}
+        href={
+          ~p"/#{@selected_project.account.name}/#{@selected_project.name}/runs/#{@command_event.id}/download"
+        }
+        label={gettext("Download result")}
+        variant="secondary"
+        size="medium"
+      >
+        <:icon_left><.download /></:icon_left>
+      </.button>
+      <.button
+        :if={Tuist.Runs.build_ci_run_url(@run)}
+        href={Tuist.Runs.build_ci_run_url(@run)}
+        label={gettext("CI Run")}
```

Comment:
If we end up adding Namespace too (which we can do in a follow-up PR) we could have a CI Environment link.